custom_components/tuya_local/devices/bresser_smart_thermo_hygrometer.yaml

```diff
@@ -0,0 +1,191 @@
+name: 7-Channel Smart Thermo Hygrometer
+products:
+  - id: xpxdr5q6vc8aztq0
+    name: Bresser Tuya Smart Thermo-Hygrometer
+primary_entity:
+  entity: sensor
+  class: temperature
+  dps:
+    - id: 1
+      name: sensor
+      type: integer
+      unit: C
+      mapping:
+        - scale: 10
+      class: measurement
+secondary_entities:
+  - entity: sensor
+    class: humidity
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: select
+    translation_key: temperature_unit
```

Comment:
Does this affect the readings coming back from the unit, or only the display?

Reply:
It only affects the display, just like with `th16_temp_humidity_sensor.yaml`, which inspired this configuration.